LLMonFHIR/FHIR Display/FHIRResourcesView.swift

```diff
@@ -7,31 +7,39 @@
 //
 
 import ModelsR4
+import SpeziLocalStorage
+import SpeziOnboarding
 import SpeziOpenAI
 import SwiftUI
 
+private enum FHIRMultipleResourceInterpreterConstants {
+        static let storageKey = "FHIRMultipleResourceInterpreter.Cache"
+    }
```

Comment:
Please only indent new lines in new scopes by 4 characters:
```suggestion
    static let storageKey = "FHIRMultipleResourceInterpreter.Cache"
}
```